server/src/libs/environment/env.ts

```diff
@@ -5,13 +5,15 @@ import { config as dotEnvConfig } from 'dotenv'
 
 import { parseBooleanVariables } from './parseBooleanVariables'
 
-const pathDevSource = join(__dirname, '../../..', `.env.${process.env.ENV}`)
+const pathDevSource = join(__dirname, '../../..', `.env.${process.env.ENV || ''}`)
 const pathBuildSource = join(__dirname, '../../../..', '.env')
 const path = existsSync(pathDevSource) ? resolve(pathDevSource) : resolve(pathBuildSource)
 const __DEV__ = pathDevSource === path
 
 if (!existsSync(path)) {
-  throw new Error(`DotEnv configuration not found for environment ${process.env.ENV} (${path})`)
+  throw new Error(
+    `DotEnv configuration not found for environment ${process.env.ENV || ''} (${path})`
+  )
```

Comment:
Pour fix `Argument of type 'string | undefined' is not assignable to parameter of type 'string'`